src/sentry/api/endpoints/project_ownership.py

```diff
@@ -79,6 +82,21 @@ def save(self):
 
         return ownership
 
+    def __modify_auto_assignment(self, ownership):
+        import logging
+
+        logger = logging.getLogger(__name__)
+        changed = False
+        logger.info(self.object)
+
+        if 'autoAssignment' in self.object:
```

Comment:
nit: its arguably simpler to just auto_assignment = self.object.get('autoAssignment') and then check if its set and does not match the current value

Reply:
Sounds indeed better, I will change this as suggested.